core/tests/unit/ldml/test_transforms.cpp

```diff
@@ -645,21 +645,30 @@ int test_strutils() {
   {
     marker_map map;
     std::cout << __FILE__ << ":" << __LINE__ << "   - bad1" << std::endl;
-    const std::u32string src = U"6\U0000ffffq"; // missing code
+    const std::u32string src = U"6\U0000ffffq"; // missing sentinel subtype
     const std::u32string dst = remove_markers(src, map);
-    const std::u32string expect = U"6q";
+    const std::u32string expect = U"6"; // 'q' removed
     zassert_string_equal(dst, expect);
     assert_equal(map.size(), 0);
   }
   {
     marker_map map;
-    std::cout << __FILE__ << ":" << __LINE__ << "   - bad1" << std::endl;
+    std::cout << __FILE__ << ":" << __LINE__ << "   - bad1b" << std::endl;
     const std::u32string src = U"6\U0000ffff"; // missing code
     const std::u32string dst = remove_markers(src, map);
     const std::u32string expect = U"6";
     zassert_string_equal(dst, expect);
     assert_equal(map.size(), 0);
   }
+  {
+    marker_map map;
+    std::cout << __FILE__ << ":" << __LINE__ << "   - bad1c" << std::endl;
+    const std::u32string src = U"6\U0000ffffzz"; // missing code
+    const std::u32string dst = remove_markers(src, map);
+    const std::u32string expect = U"6z";
+    zassert_string_equal(dst, expect);
+    assert_equal(map.size(), 0);
```

Comment:
I think given our discussion today, this is something that should never appear in keyboard processor inputs so we shouldn't need to cater to it, right?

Reply:
It was for robustness, but I think what we converged on was to leave it unchanged.